services/src/main/scala/cromwell/services/keyvalue/KeyValueServiceActor.scala

```diff
@@ -49,3 +60,22 @@ trait KeyValueServiceActor extends Actor {
     }
   }
 }
+
+/**
+  * Perhaps not great in production, but fine for test cases...!
```

Comment:
Looks like sarcasm doesn't work well in text form. Actually I just tried this and I'm getting issues with test-only artifacts not being picked up between different subprojects. IMO leaving it in `services/main` is better than copy/pasting it between multiple subprojects (sfsBackend, jesBackend) but happy to do that if you prefer?